README.md

```diff
@@ -73,10 +78,16 @@ end
 
 In this example, we are running a multivariate test with 3 options being
 tested: the existing version (control), and two title changes. The minimum
-number of variants in any test should be two.
+number of variants in any test should be two. 
+
+### 2. HTTP response header to fastly
+
+The `configure_response` method used in the example in `step 1` sends the response header. The header helps Fastly to understand which variant was returned to the user and cache appropriately.
 
-Then, add this to your layouts, so that we have a meta tag that can be picked up
-by the extension and analytics.
+### 3. 
```

Comment:
In the other numbers you included a heading alongside the number. Could we put something here? Maybe `Add this to your layouts` and then re-word the content below?